src/components/Accordion/Accordion.tsx

```diff
@@ -1,69 +1,121 @@
-  /** Inner label on the button that opens the accordion */
-  accordionLabel?: React.ReactNode;
-  /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
-  blockName?: string;
-  /** className you can add in addition to 'input' */
-  className?: string;
-  /** Whether the accordion is open by default */
-  defaultOpen?: boolean;
+  /** Array of data to display */
+  contentData: AccordionContentDataProps[];
   /** ID that other components can cross reference for accessibility purposes */
   id?: string;
-  /** accordionLabel's input ID */
-  inputId?: string;
-  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
-  modifiers?: string[];
+  /** Whether the accordion is open by default only on its initial rendering */
+  isDefaultOpen?: boolean;
+  /** Whether the accordion should display with a fixed height and have
+   * the panel scrollable. */
+  isScrollable?: boolean;
 }
 
-/** Accordion component that shows content on toggle */
-export default function Accordion(
-  props: React.PropsWithChildren<AccordionProps>
-) {
+/**
+ * Get the minus or plus icon depending on whether the accordion
+ * is open or closed.
+ */
+const getIcon = (isExpanded = false) => {
+  const iconName = isExpanded ? IconNames.minus : IconNames.plus;
+  return <Icon decorative={true} name={iconName} modifiers={["small"]} />;
+};
+
+/**
+ * Returns `AccordionItems` for every accordion object in the data
+ * array. This automatically creates the `AccordionButton` and `AccordionPanel`
+ * combination that is required for the Chakra `Accordion` component.
+ */
+const getElementsFromContentData = (data = [], scroll) => {
+  // For FAQ-style multiple accordions, the button should be bigger.
+  // Otherwise, use the default.
+  const multiplePadding = data?.length > 1 ? 4 : null;
+  const panelScrollStyle = scroll
+    ? ({
+        maxHeight: "inherit",
+        overflowY: "auto",
+      } as any)
+    : {};
+
+  return data.map((content, index) => {
+    // This is done to support both string and DOM element input.
+    const panel =
+      typeof content.panel === "string" ? (
+        <AccordionPanel
+          key={index}
+          dangerouslySetInnerHTML={{ __html: content.panel }}
+        />
+      ) : (
+        <AccordionPanel key={index}>{content.panel}</AccordionPanel>
+      );
+
+    return (
+      <AccordionItem key={index} maxHeight={scroll}>
+        {/* Get the current state to render the correct icon. */}
+        {({ isExpanded }) => (
+          <>
+            <AccordionButton padding={multiplePadding}>
+              <Box flex="1" textAlign="left">
+                {content.label}
+              </Box>
+              {getIcon(isExpanded)}
+            </AccordionButton>
+            <Box
+              {...{
+                ...panelScrollStyle,
+                maxHeight: isExpanded && scroll ? "200px" : undefined,
```

Comment:
TBH this `"200px"` value was arbitrary but it's the one that worked and looked the best. Otherwise... the way Chakra composes its own panel is hard to style.

Reply:
Do we even still need this scrollable feature variant?

Reply:
@EdwinGuzman  We do not need this variant.  I spoke to Kathy about this and a fixed height option is not being used anywhere.

Reply:
If this is functional, let's leave it in there.  That being said, I actually think a prop for setting `maxHeight` would be more beneficial than this.  Not needed, but more apropos.

Reply:
It might be functional if we decide that it's okay to have a lot of content to render. If the UX for having to scroll down and up the page for a long accordion section, then we can just remove this variant. Or, we can recommend that content that goes in an Accordion should not exceed a certain length.

Reply:
OK, let's not forget about this.